### PR TITLE
Linux-CSMA-Opentelemetry docs

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+title: Linux - Cloud Security Monitoring and Analytics - OpenTelemetry
+dashboard: The Sumo Logic app for Linux Cloud Security Monitoring and Analytics - OpenTelemetry provides better understanding of your production environments, and surface relevant insights by tuning out-of-the-box content to align with your security team’s focus
+
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/cloud-security-monitoring-analytics/SecMon_Linux.png')} alt="Thumbnail icon" width="100"/>
+
+The Linux - Cloud Security Monitoring and Analytics - OpenTelemetry is a unified log app that ingests distribution of Linux data to Sumo Logic via OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver). The app's preconfigured dashboards provide insight into User, Service, System Monitoring, Login, and Privilege Activity, which provides a better understanding of your production environments and surface relevant insights by tuning out-of-the-box content to align with your security team's focus.
```
Comment (kimsauce) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:12]:
```suggestion
The Linux - Cloud Security Monitoring and Analytics - OpenTelemetry is a unified log app that ingests distribution of Linux data to Sumo Logic via OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver). The app's preconfigured dashboards provide insight into user, service, systems, login, and privilege activity, providing a better understanding of your production environments and surface relevant insights by tuning out-of-the-box content to align with your security team's focus.
```

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+:::note
+By default, the path for Linux log files required for all the distros are pre-populated in the UI. (Optional) Unwanted file paths can be removed from the list if the files are not available on your Linux distribution. The collection will work even if not all the files are present in your system.
+:::
+
+### Step 3: Send logs to Sumo Logic
+
+{@import ../../../reuse/apps/opentelemetry/send-logs-intro.md}
+
+1.  Copy the YAML file to `/etc/otelcol-sumo/conf.d/` folder in the Linux instance which needs to be monitored.
+2.  Restart the collector using:
```
Comment (kimsauce) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:70]:
```suggestion
1. Copy the YAML file to `/etc/otelcol-sumo/conf.d/` folder in the Linux instance which needs to be monitored.
2. Restart the collector using:
```

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+| sort by %"LatestTime" 
+```
+
+## Viewing Linux - Cloud Security Monitoring and Analytics dashboards
+
+{@import ../../../reuse/filter-dashboards.md}
+
+### Overview
+
+The **Linux - Security Analytics - Overview** dashboard provides an overview of security statistics relevant for Linux systems and presents information about successful and failed logins, root login successes and failures, user accounts created and deleted, sudo attempts, and total Services.
```
Comment (kimsauce) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:106]:
```suggestion
The **Linux - Security Analytics - Overview** dashboard provides an overview of security statistics relevant for Linux systems and presents information about successful and failed logins, root login successes and failures, user accounts created and deleted, sudo attempts, and total services.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Great job!

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
+---
+id: linux-opentelemetry
+title: Linux - Cloud Security Monitoring and Analytics - OpenTelemetry
+dashboard: The Sumo Logic app for Linux Cloud Security Monitoring and Analytics - OpenTelemetry provides better understanding of your production environments, and surface relevant insights by tuning out-of-the-box content to align with your security team’s focus
+
```
Comment (kimsauce) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:5]:
```suggestion
```

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/cloud-security-monitoring-analytics/SecMon_Linux.png')} alt="Thumbnail icon" width="100"/>
+
+The Linux - Cloud Security Monitoring and Analytics - OpenTelemetry is a unified log app that ingests distribution of Linux data to Sumo Logic via OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver). The app's preconfigured dashboards provide insight into user, service, systems, login, and privilege activity, providing a better understanding of your production environments and surface relevant insights by tuning out-of-the-box content to align with your security team's focus.
```
Comment (kimsauce) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:12]:
```suggestion
Linux - Cloud Security Monitoring and Analytics - OpenTelemetry is a unified log app that ingests distribution of Linux data to Sumo Logic via OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver). The app's preconfigured dashboards provide insight into user, service, systems, login, and privilege activity, providing a better understanding of your production environments and surface relevant insights by tuning out-of-the-box content to align with your security team's focus.
```

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/PCI-Compliance-For-Linux/OpenTelemetry/Linux-Collector.png' alt="Linux-Collector" style={{border: '1px solid black'}} />
+
+### Step 2: Configure integration
+
+In this step, you will configure the yaml required for Linux Collection. The app requires path for system log file based on the Linux version used.
+
+#### Required Logs for Ubuntu
+
+The following logs, located in the `/var/log` folder, are required for using the Sumo Logic app for PCI compliance for Linux with Ubuntu.
```
Comment (kimsauce) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:40]: This refers to a different app altogether, PCI Compliance for Linux. Is this intentional?

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+- auth.log
+- syslog
+- daemon.log
+- dpkg.log
+- kern.log
+
+#### Required Logs for CentOS, Amazon Linux, and Red Hat
+
+The following logs, located in the `/var/log` folder, are required for using the Sumo Logic app for PCI compliance for Linux with CentOS, Amazon Linux, and Red Hat.
```
Comment (kimsauce) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:50]: see https://github.com/SumoLogic/sumologic-documentation/pull/2707/files#r1250230853

**Review by kimsauce — DISMISSED** [review]
minor stuff

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md (hunk 6 above)
Comment (JV0812) [reply docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:40]: Yes Kim, it is intentional. You can refer this [doc](https://help.sumologic.com/docs/integrations/pci-compliance/opentelemetry/linux-opentelemetry/#step-1-set-up-collector).

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md (hunk 7 above)
Comment (JV0812) [reply docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:49]: Yes Kim, it is intentional. You can refer this [doc](https://help.sumologic.com/docs/integrations/pci-compliance/opentelemetry/linux-opentelemetry/#step-1-set-up-collector).

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+
+### Step 1: Set up Collector
+
+{@import ../../../reuse/apps/opentelemetry/set-up-collector.md}
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/PCI-Compliance-For-Linux/OpenTelemetry/Linux-Collector.png' alt="Linux-Collector" style={{border: '1px solid black'}} />
+
+### Step 2: Configure integration
+
+In this step, you will configure the yaml required for Linux Collection. The app requires path for system log file based on the Linux version used.
```
Comment (kimsauce) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:35]:
```suggestion
In this step, you will configure the yaml required for Linux collection. The app requires path for system log file based on the Linux version used.
```

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md (hunk 6 above)
Comment (kimsauce) [reply docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:39]: Shouldn't this be reflected in this doc's title? It says it's called "Linux - Cloud Security Monitoring and Analytics - OpenTelemetry".

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+
+<img src={useBaseUrl('img/integrations/cloud-security-monitoring-analytics/SecMon_Linux.png')} alt="Thumbnail icon" width="100"/>
+
+Linux - Cloud Security Monitoring and Analytics - OpenTelemetry is a unified log app that ingests distribution of Linux data to Sumo Logic via OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver). The app's preconfigured dashboards provide insight into user, service, systems, login, and privilege activity, providing a better understanding of your production environments and surface relevant insights by tuning out-of-the-box content to align with your security team's focus.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/PCI-Compliance-For-Linux/OpenTelemetry/Linux-Schematics.png' alt="Linux-Schematics" style={{border: '1px solid black'}} />
+
+## Fields created in Sumo Logic for Linux - Security Analytics
+
+Following tag will be created as part of Linux app installation, if not already present.
```
Comment (jpipkin1) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:17]:
```suggestion
The following tag will be created as part of Linux app installation, if not already present. 
```

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+
+- `sumo.datasource`. Has a fixed value of **linux**.
+
+## Collection configuration and app installation
+
+:::note
+You can skip this section if you have already set up the logs collection through [Linux](/docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry/) or [Linux PCI](/docs/integrations/pci-compliance/opentelemetry/linux-opentelemetry) app installation. Additional collection is not required as the logs used by this app are already ingested into Sumo Logic.
+:::
+
+### Step 1: Set up Collector
```
Comment (jpipkin1) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:27]:
```suggestion
### Step 1: Set up collector
```

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+
+{@import ../../../reuse/apps/opentelemetry/set-up-collector.md}
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/PCI-Compliance-For-Linux/OpenTelemetry/Linux-Collector.png' alt="Linux-Collector" style={{border: '1px solid black'}} />
+
+### Step 2: Configure integration
+
+In this step, you will configure the yaml required for Linux collection. The app requires path for system log file based on the Linux version used.
```
Comment (jpipkin1) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:35]:
```suggestion
In this step, you will configure the YAML required for Linux collection. The app requires path for system log file based on the Linux version used.
```

File: docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+#### Required Logs for CentOS, Amazon Linux, and Red Hat
+
+The following logs, located in the `/var/log` folder, are required for using the Sumo Logic app for Cloud Security Monitoring and Analytics for Linux with CentOS, Amazon Linux, and Red Hat.
+
+- audit/audit.log
+- secure
+- Messages
+- yum.log 
+
+Click on the **Download YAML File** button to get the yaml file.
```
Comment (jpipkin1) [inline docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md:56]:
```suggestion
Click on the **Download YAML File** button to get the YAML file.
```

**Review by jpipkin1 — APPROVED** [review]
This looks great, @JV0812! Approved. I just suggested a few small updates.

**Comment by JV0812** [comment]
Thanks John, I have fixed your comments.

**Pull request merged**